Geometry/MTDCommonData/src/ETLNumberingScheme.cc

```diff
@@ -0,0 +1,110 @@
+#include "Geometry/MTDCommonData/interface/ETLNumberingScheme.h"
+#include "DataFormats/ForwardDetId/interface/ETLDetId.h"
+
+#include <iostream>
+
+//#define EDM_ML_DEBUG
+
+ETLNumberingScheme::ETLNumberingScheme() : 
+  MTDNumberingScheme() {
+  edm::LogInfo("MTDGeom") << "Creating ETLNumberingScheme";
+}
+
+ETLNumberingScheme::~ETLNumberingScheme() {
+  edm::LogInfo("MTDGeom") << "Deleting ETLNumberingScheme";
```

Comment:
I would left LogInfo in a destructor under DEBUG regime only

Reply:
I agree it is not compelling, I will add an ifdef. Although we are having the same code in the Ecal numbering scheme since 12 years (this is basically a clone of that), and in the Muon numbering scheme constructor, even worse, we have couts. So plenty of things one may improve...